### ISSUE TITLE
Crashes From SDK on Android

### ISSUE THREAD

**Issue by jjmutumi** [UseHover/SDK-Issues#48]
## Expected Behavior
The app should not crash

## Current Behavior
Crashes in production with report sent via crashlytics

## Possible Solution
--

## Steps to Reproduce
Have not experienced it personally in dev or production

## Context (Environment)
Using release:  1.6.1

Some devices experiencing it:

Brand: LGE
Model: Nexus 4
Orientation: Portrait
RAM free: 622.91 MB
Disk free: 2.83 GB
Operating System
Version: 4.1.2
Orientation: Portrait
Rooted: Yes

## Detailed Description
```
Fatal Exception: java.lang.RuntimeException
Unable to start activity ComponentInfo{com.opareta.opareta/com.hover.sdk.permissions.PermissionActivity}: android.view.InflateException: Binary XML file line #15: Error inflating class TextView

Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.opareta.opareta/com.hover.sdk.permissions.PermissionActivity}: android.view.InflateException: Binary XML file line #15: Error inflating class TextView
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2059)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084)
       at android.app.ActivityThread.access$600(ActivityThread.java:130)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at com.android.qihooSandbox.QihooBridge.invokeOriginalMethodNative(QihooBridge.java)
       at com.android.qihooSandbox.QihooBridge.handleHookedMethod(QihooBridge.java:840)
       at android.os.Handler.dispatchMessage(Handler.java)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4745)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)

Caused by android.view.InflateException
Binary XML file line #15: Error inflating class TextView
android.view.LayoutInflater.createViewFromTag

Caused by android.content.res.Resources$NotFoundException
Resource is not a Drawable (color or path): TypedValue{t=0x1/d=0x7f070195 a=-1 r=0x7f070195}
com.hover.sdk.permissions.PermissionActivity.onCreate
```

**Comment by yveskalume** [comment]
I have the same problem did you find the solution ?

**Comment by jjmutumi** [comment]
At the time this was fixed by upgrading SDK to 1.6.3
